src/themes/default/components/core/Logo.vue

```diff
@@ -4,13 +4,20 @@
       :width="width"
       :height="height"
       src="/assets/logo.svg"
-      alt="Vuestore logo"
+      :alt="$t(appName)"
```

Comment:
I don't think that we need translating the store name. Could we instead add a possibility to configure the store name on storeview basis as well (in multistore mode)?

Reply:
- Yeah sure, Nice to have store name base on selection
 I'll do in the same request.